app/src/main/java/com/gmail/mxwild/newhabit/moviedetail/MovieDetailsViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.gmail.mxwild.newhabit.moviedetail
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.gmail.mxwild.newhabit.model.data.Movie
+
+class MovieDetailsViewModel(private val movie: Movie) : ViewModel() {
```

Comment:
Why to have such view model at all? What is the benefit of it?

Reply:
Reformat